generator.py

```diff
@@ -1,19 +1,23 @@
 # 2. написати свій генератор. Приймає int (кількість слів) а повертає рандомні слова.
 #    Слова мають бути унікальні. Max int = 10_000.
 
-
 from faker import Faker
 
 
 def word_generator(word_count):
     if word_count > 10000:
-        print("Error: Max value not more then 10,000")
+        print("Error: Max value not more than 10,000")
         return
     fake = Faker()
-    for _ in range(word_count):
-        yield fake.word()
+    generated_words = set()
+
+    while len(generated_words) < word_count:
```

Comment:
Did you test this code? Check it with value like 5000 not 500. And check performance. 
Really bad code here, just system of guessing ))